### ISSUE TITLE
Ignore empty title attributes

### ISSUE THREAD

**Issue by winniehell** [gettalong/kramdown#199]
An image with explicitly specified empty title attribute leads to incorrect Markdown:
```
$ kramdown -ihtml -okramdown <<<'<img src="x" alt="" title="" />'
![](x "")
```

As mentioned [here](http://kramdown.gettalong.org/syntax.html#inline-links) the title must not be empty in Markdown and passing the output back to kramdown leads to strange behavior:
```
$ kramdown <<<'![](x "")'
<p>![](x “”)</p>
```

Please ignore empty title attributes as it already happens for links:
```
$ kramdown -ihtml -okramdown <<<'<a href="x" title="">y</a>'
[y](x)
```

**Comment by winniehell** [comment]
Sorry for all the spam... seems like GitHub keeps the references from issues to commits even though the commits themselves are not referenced anymore. :confused:

**Comment by gettalong** [comment]
Fixed by your pull request #206 - thanks!
